src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -249,22 +324,20 @@ impl<'a, 'tcx> Qualifier<'a, 'tcx, 'tcx> {
                                    self.mir.local_kind(index) == LocalKind::Arg) &&
                                    self.tcx.sess.features_untracked().const_let => {
                 debug!("store to var {:?}", index);
-                self.local_qualif[index] = Some(self.qualif);
```

Comment:
Hmm you're losing the distinction between uninitialized and no flags set.

Reply:
Yep. What do you recommend? You're the one who suggested moving to `IdxSetBuf` instead of `BitVector` if I remember...